service/src/main/java/uk/nhs/adaptors/gpc/consumer/filters/SslContextBuilderWrapper.java

```diff
@@ -1,52 +1,36 @@
 package uk.nhs.adaptors.gpc.consumer.filters;
 
 import java.security.KeyStore;
-import java.util.ArrayList;
 import java.util.UUID;
 
 import javax.net.ssl.KeyManagerFactory;
 import javax.net.ssl.TrustManagerFactory;
 
-import org.apache.commons.lang3.StringUtils;
-
 import com.heroku.sdk.EnvKeyStore;
 
 import io.netty.handler.ssl.SslContext;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
-import uk.nhs.adaptors.gpc.consumer.gpc.exception.GpConnectException;
+import org.springframework.beans.factory.annotation.Value;
 import uk.nhs.adaptors.gpc.consumer.utils.PemFormatter;
 
+
 @Slf4j
 public class SslContextBuilderWrapper {
```

Comment:
Hmm, would injecting `GpcConfiguration` not be simpler?

Reply:
Spoken on a teams call about the reasoning for this, but it would introduce a whole load of complexity / refactoring to work by injecting the configuration class directly.